docs/application/dotnet/guides/nui/text.md

```diff
@@ -0,0 +1,574 @@
+
+    The following examples show text in red color:
+
+    ```csharp
+    label.Text = "<color value='red'>Red Text</color>"; /// Color coded with a text constant
+    ```
+
+    ```csharp
+    label.Text = "<color value='0xFFFF0000'>Red Text</color>"); /// Color packed inside an ARGB hexadecimal value
+    ```
+
+-   `<font>`
+
+    Sets the font values for the characters inside the element.
+
+    The following attributes are supported:
+
+    -   `family`: Font name
+    -   `size`: Font size in points
+    -   `weight`: Font weight
+    -   `width`: Font width
+    -   `slant`: Font slant
+
+    For more information on the attribute values, see [Select Font](#font).
+
+    The following example sets the font family and weight:
+
+    ```csharp
+    label.Text = "<font family='SamsungSans' weight='bold'>Hello world</font>";
+    ```
+
+
+
+<a name="textField"></a>
+## TextField
+
+The `TextField` class provides a control that allows single line editable text field.
+
+**Figure: TextField**
+
+![TextField](./media/textfield.png)
+
+
+<a name="textField1"></a>
+### TextField Events
+
+The following table lists the basic signals provided by the `TextField` class:
+
+
+| Input signal        | Description                                 |
+| ------------------- | ------------------------------------------- |
+| `TextChanged`       | Emitted when the text changes.              |
+| `MaxLengthReached`  | Emitted when the inserted text exceeds the maximum character limit. |
+
+<a name="textField2"></a>
+### Create TextField
+
+Before the text is entered, the `TextField` class displays a placeholder text. An alternative placeholder is displayed when `TextField` gets the keyboard focus. For example, the `TextField` that is used to enter a username initially displays the text `Unknown Name` and then the text `Enter Name`, when the cursor is visible.
+
+The following example illustrates the creation of a `TextField` object:
+
+```csharp
+Window window = Window.Instance;
+TextField field = new TextField();
+PropertyMap propertyMap = new PropertyMap();
+propertyMap.Add("placeholderText", new PropertyValue("Unknown Name"));
+propertyMap.Add("placeholderTextFocused", new PropertyValue("Enter Name."));
+field.Placeholder = propertyMap;
+window.Add(field);
+```
+
+When the `TextField` is tapped, it automatically gets the keyboard focus. Key events correspond to entering the text. Additionally, the placeholder text is removed as soon as the text is entered. The text entered can be retrieved by using the `TEXT` property:
+
+```csharp
+Property::Value fieldText = field.GetProperty( TextField::Property::TEXT );
+std::string fieldTextString = fieldText.Get< std::string >();
+```
+
+<a name="textField3"></a>
+### Align Text

```

Comment:
### Align Text of a TextField

Reply:
Referencing https://github.com/Samsung/tizen-docs/pull/1009#discussion_r398493508
I add the updated suggestion

\### Align Text in TextField